atdgen/src/ox_emit.mli

```diff
@@ -37,17 +37,17 @@ val write_ocaml : target -> string -> string -> unit
 
 val name_of_var : string -> string
 
-val nth : Mapping.loc_id -> int -> int -> Mapping.loc_id
+val nth : string -> int -> int -> string
```

Comment:
I was wondering why you didn't like having a type alias `loc_id = string`. The original intent was to document function signatures without the burdening ourselves with an abstract type.